rules/96.5g_http2_dos_recognition.xml

```diff
@@ -0,0 +1,68 @@
+<beginning>
+
+
+
+
+
+
+<embedded_functions><![CDATA[
+#include <stdlib.h>
+#include <mmt_core.h>
+
+
+#include<math.h>
+
+static unsigned long int limit_5g_dos_http2_initialuemessage_per_ms = 0;
+
+void on_load(){
+	const char *str = getenv("MMT_SEC_5G_DOS_NGAP_INITIALUEMESSAGE_MS_LIMIT");
+	if( str == NULL ){
+		mmt_info("Rule 96: no value of MMT_SEC_5G_DOS_NGAP_INITIALUEMESSAGE_MS_LIMIT" );
```

Comment:
`NGAP_INITIALUEMESSAGE` is not applicable

Reply:
I updated with the correct message